tests/FakeItEasy.Tests.netstd/project.json

```diff
@@ -1,13 +1,7 @@
-{
+﻿{

```

Comment:
FYI this is an encoding change from ASCII to UTF8-BOM, inline with Visual Studio's preference and reflected in our `.editorconfig`.